js/script.js

```diff
@@ -1,2 +1,25 @@
-let p = document.querySelector ("p")
-p.innerHTML = "Hello"
+document.querySelector('#push').onclick = function(){
```

Comment:
Read about event listeners.
They can help you understand this approach to function assignments better.
For more info on this refer to - https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener